### PR TITLE
Added Microsoft Azure, Updated to 1.12 GA

### PR DESCRIPTION
**Pull request by derdanu**

### CODE REVIEW

**Comment by londoncalling** [comment]
LGTM, cc'ing @friism

File: swarm-mode/beginner/swarm.sh [hunk 1]
```diff
@@ -79,14 +79,18 @@ while [ "$#" -gt 0 ]; do
...
       usage
       ;;
   esac
 done
 
-# Value of driver parameter's value must be among "digitalocean", "amazonec2", "virtualbox" (if no value is provided, "virtualbox" driver is used)
-if [ "$DRIVER" != "virtualbox" -a "$DRIVER" != "digitalocean" -a "$DRIVER" != "amazonec2" ];then
+# Value of driver parameter's value must be among "azure", "digitalocean", "amazonec2", "virtualbox" (if no value is provided, "virtualbox" driver is used)
+if [ "$DRIVER" != "virtualbox" -a "$DRIVER" != "digitalocean" -a "$DRIVER" != "amazonec2"  -a "$DRIVER" != "azure" ];then
   error "driver value must be among digitalocean, amazonec2, virtualbox"
```
Comment (ManoMarks) [inline swarm-mode/beginner/swarm.sh]: Should probably add Azure to the error message too

**Comment by ManoMarks** [comment]
Please also sign the PR as outlined here: https://github.com/docker/docker/blob/master/CONTRIBUTING.md#sign-your-work

**Comment by ManoMarks** [comment]
@derdanu can you sign the PR too?

**Comment by derdanu** [comment]
Added azure to the error message and signed all commits.

**Pull request merged**